code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	lips
+		name = "Lips"
+		icon_state = "lips"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	lipcorner_left
+		name = "Lip Corner Left"
+		icon_state = "lipcorner_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	lipcorner_right
+		name = "Lip Corner Right"
+		icon_state = "lipcorner_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	neck
+		name = "Neck Cover"
+		icon_state = "neck"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	neckthick
+		name = "Neck Cover (Thick)"
+		icon_state = "neckthick"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	nosestripe
+		name = "Nose Stripe"
+		icon_state = "nosestripe"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	nosetape
+		name = "Nose Tape"
+		icon_state = "nosetape"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Vaurca")
+
+	scratch_abdomen_left
+		name = "Scratch, Abdomen Left"
+		icon_state = "scratch_abdomen_l"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_abdomen_right
+		name = "Scratch, Abdomen Right"
+		icon_state = "scratch_abdomen_r"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_abdomen_small_left
+		name = "Scratch, Abdomen Small Left"
+		icon_state = "scratch_abdomensmall_l"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_abdomen_small_right
+		name = "Scratch, Abdomen Small Right"
+		icon_state = "scratch_abdomensmall_r"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_back
+		name = "Scratch, Back"
+		icon_state = "scratch_back"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi", "Vaurca")
+
+	scratch_chest_left
+		name = "Scratch, Chest (Left)"
+		icon_state = "scratch_chest_l"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	scratch_chest_right
+		name = "Scratch, Chest (Right)"
+		icon_state = "scratch_chest_r"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	skull_paint
+		name = "Skull Paint"
```

Comment:
This needs to be removed, it makes zero sense in the role play environment of the station

Reply:
Can you actually explain why.

Reply:
Full face paint makes no sense in a working enviroment